packages/commons/tests/unit/config/EnvironmentVariablesService.test.ts

```diff
@@ -0,0 +1,88 @@
+/**
+ * Test EnvironmentVariablesService class
+ *
+ * @group unit/commons/all
+ */
+
+import { EnvironmentVariablesService } from '../../../src/config';
+
+describe('Class: EnvironmentVariablesService', () => {
+
+  const ENVIRONMENT_VARIABLES = process.env;
+
+  beforeEach(() => {
+    jest.resetModules();
+    process.env = { ...ENVIRONMENT_VARIABLES };
+  });
+
+  afterAll(() => {
+    process.env = ENVIRONMENT_VARIABLES;
+  });
+
+  describe('Method: get', () => {
+
+    test('When the variable IS present, it returns the value of a runtime variable', () => {
+
+      // Prepare
+      process.env.CUSTOM_VARIABLE = 'my custom value';
+      const service = new EnvironmentVariablesService();
+
+      // Act
+      const value = service.get('CUSTOM_VARIABLE');
+
+      // Assess
+      expect(value).toEqual('my custom value');
+
+    });
+
+    test('When the variable IS NOT present, it returns an empty string', () => {
+
+      // Prepare
+      delete process.env.CUSTOM_VARIABLE;
+      const service = new EnvironmentVariablesService();
+
+      // Act
+      const value = service.get('CUSTOM_VARIABLE');
+
+      // Assess
+      expect(value).toEqual('');
+
+    });
+
+  });
+
+  describe('Method: getServiceName', () => {
+
+    test('It returns the value of the environment variable POWERTOOLS_SERVICE_NAME', () => {
+
+      // Prepare
+      process.env.POWERTOOLS_SERVICE_NAME = 'shopping-cart-api';
+      const service = new EnvironmentVariablesService();
+
+      // Act
+      const value = service.getServiceName();
+
+      // Assess
+      expect(value).toEqual('shopping-cart-api');
+    });
+
+  });
+
+  describe('Method: getXrayTraceId', () => {
+
+    test('It returns the value of the environment variable _X_AMZN_TRACE_ID', () => {
```

Comment:
can we add better test for this logic: 
1. if empty ?
2. if real (see the split works)